.github/workflows/publish-extensions.yml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches:
       - 'main'
+    tags:
+      - "v*"
```

Comment:
Since the extensions will now publish based off a new tag being committed, this needs to be added to ensure the workflow runs on new tags.